requirements.txt

```diff
@@ -1,4 +1,5 @@
 bcrypt==3.1.3
+cryptography==1.7.1
```

Comment:
For some reason my builds are failing whenever I try to use a different version from 1.7.1

I'll keep looking into it but I don't really understand why that's happening